visualization/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 ### Fixed 🐞
 
 - Fix issue where zooming out too much makes the map disappear and zooming in too much causes you to go through the map. [#3697](https://github.com/MaibornWolff/codecharta/pull/3697)
+- Camera perspective is correctly adopted from the custom configuration[#3194](https://github.com/MaibornWolff/codecharta/pull/3194)
```

Comment:
Not so sure why, but this redirects to an issue. According to our guidelines it should be a PR, preferably this one.